site/gatsby-site/content/about/index.es.mdx

```diff
@@ -41,9 +41,60 @@ La base de datos de incidentes es administrada de manera participativa por perso
 **Contribuciones:** Patrick es el colaborador [principal](https://incidentdatabase.ai/summaries/leaderboard) de informes de incidentes para el Proyecto de base de datos de incidentes de AI.
 
-* **[Sean McGregor](https://seanbmcgregor.com/):** Sean McGregor fundó AI Incident Da
\ No newline at end of file
+* **[Sean McGregor](https://seanbmcgregor.com/):** Sean McGregor fundó el proyecto de la base de datos de incidentes de IA y recientemente dejó un puesto como arquitecto de aprendizaje automático en la startup del acelerador neuronal Syntiant para poder concentrarse en la seguridad. de sistemas inteligentes a tiempo completo. El trabajo del Dr. McGregor abarca aceleradores neuronales para la inferencia de eficiencia energética, aprendizaje profundo para el habla y la heliofísica, y aprendizaje de refuerzo para la política de supresión de incendios forestales. Fuera de su trabajo remunerado, Sean organizó una serie de talleres en las principales conferencias académicas de IA sobre el tema "AI for Good" y actualmente está desarrollando un enfoque basado en incentivos para hacer que la IA sea más segura a través de auditorías y seguros. **Contribuciones:** Sean es voluntario como mantenedor de proyectos y editor del proyecto Base de datos de incidentes de IA (AIID).
+
+**Miembros sin derecho a voto**
+
+* **[Neama Dadkhahnikoo](https://www.linkedin.com/in/neama/):** Neama Dadkhahnikoo es experta en inteligencia artificial y emprendimiento, con más de 15 años de experiencia en desarrollo de tecnología en startups, no beneficios y grandes empresas. Actualmente se desempeña como gerente de productos para Vertex AI, la plataforma unificada de Google Cloud para el aprendizaje automático de extremo a extremo. Anteriormente, el Sr. Dadkhahnikoo fue director de IA y operaciones de datos en la Fundación XPRIZE, CTO de CaregiversDirect (inicio de IA para atención domiciliaria), cofundador y director de operaciones de Textpert (inicio de IA para salud mental) y consultor de inicio. Comenzó su carrera como desarrollador de software para The Boeing Company. El Sr. Dadkhahnikoo tiene un MBA de UCLA Anderson; una Maestría en Gestión de Proyectos del Stevens Institute of Technology; y una licenciatura en Matemáticas Aplicadas y Ciencias de la Computación, con especialización en Física, de UC Berkeley.
+**Contribuciones:** Neama se ofrece como voluntario para mantener organizada la Colaboración de IA responsable fuera de su rol como gerente de producto en Google.
```

Comment:
We should keep the original English name of the project `Responsible AI Collaborative`